src/OrchardCore/OrchardCore.ContentTypes.Abstractions/Events/ContentDefinitionBuildingContextBase.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace OrchardCore.ContentTypes.Events;
+
+public abstract class ContentDefinitionBuildingContextBase
+{
+    public readonly string Name;
```

Comment:
It doesn't seem we need this. I could only find a usage testing if it is `nameof(DashboardPart)` but I feel like this is already available in the "Record" passed with each context class.

Reply:
Also, in the case of a `ContentTypePartContextBuilding` (and similar), it's not obvious what `Name` can represent. The name of the type? the part one?